From a9d5ced1ff2a58014dc850ece702cc194884870d Mon Sep 17 00:00:00 2001
From: Hector Martin <marcan@marcan.st>
Date: Sat, 12 Mar 2022 00:07:09 +0900
Subject: [PATCH 059/171] of: Demote "Bad cell count" to debug message

This happens on the SPMI bus... TODO: figure out what the right solution
is here.

Signed-off-by: Hector Martin <marcan@marcan.st>
---
 drivers/of/address.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/address.c b/drivers/of/address.c
index 94f017d808c4..68f54ec92496 100644
--- a/drivers/of/address.c
+++ b/drivers/of/address.c
@@ -538,7 +538,7 @@ static u64 __of_translate_address(struct device_node *dev,
 		pbus = of_match_bus(parent);
 		pbus->count_cells(dev, &pna, &pns);
 		if (!OF_CHECK_COUNTS(pna, pns)) {
-			pr_err("Bad cell count for %pOF\n", dev);
+			pr_debug("Bad cell count for %pOF\n", dev);
 			break;
 		}
 
-- 
2.34.1

